lib/takeaway.rb

```diff
@@ -0,0 +1,19 @@
+require 'menu.rb'
+require 'order.rb'
```

Comment:
You probably don't need all these `require`s here, because any program/interface/test suite using the classes should load them itself. Then you can control precisely which ones it loads.